libraries/license-info.json

```diff
@@ -56,5 +56,8 @@
     "license": "ISC",
     "year": "2016,",
     "fullname": "Jake Archibald <jaffathecake@gmail.com>"
+  },
+  "Roboto": {
+    "license": "Apache-2.0"
```

Comment:
Should the extra license info be filled out?

Reply:
I'll add it. I thought it wasn't necessary because a few other libraries use the same license and they don't include the info, but there's indeed a placeholder near the end of the license text:
![image](https://github.com/user-attachments/assets/ebd9ad99-a142-4d5a-bf53-867b4b4fe51a)